modules/apps/export-import/export-import-web/src/main/resources/META-INF/resources/export/new_export/export_layouts.jsp

```diff
@@ -128,6 +128,11 @@ renderResponse.setTitle(!configuredExport ? LanguageUtil.get(request, "new-custo
 		<liferay-ui:error exception="<%= LARFileNameException.class %>" message="please-enter-a-file-with-a-valid-file-name" />
 
 		<div class="export-dialog-tree">
+		<div class="alert alert-warning">
```

Comment:
Looks like it's a duplicate here.

Reply:
@thabet91 how about this? <div class="alert alert-warning"> is duplicated here. Please help double-check. Thanks.

Reply:
@tototrinh fixed